README.md

```diff
@@ -74,3 +74,39 @@ loader = tfrecord.tfrecord_loader("/path/to/data.tfrecord", None, {
 for record in loader:
     print(record["label"])
 ```
+
+### Transforming input
+
+There are use cases when you might not want to return your input directly after reading from the tfrecord files.
+A trivial example could be to normalize your input, if you haven't saved the normalized feature in the files.
+
+You can easily achieve this by leveraging the `transform_func` argument.
+There might also be use cases to remove a field after performing a transformation, this can be leveraged using the `removed_fields`.
+
+Both the arguments mentioned above are available in all readers including `tfrecord.tfrecord_loader` and `tfrecord.torch.TFRecordDataset`.
```

Comment:
It's too verbose. Change to this: "You can optionally pass a function as `transform` argument to perform post processing of features before returning. This can for example be used to decode images or normalize colors to a certain range.

Reply:
Change it, and let the removed_fields there be for now until we [resolve](https://github.com/vahidk/tfrecord/pull/31#discussion_r396194827).